test/rate-limit-policies.yml

```diff
@@ -0,0 +1,2 @@
+total-certs:
+  threshold: 10000
```

Comment:
This was pulled out of my butt, and originally it was for 24 hours, so for 90 days it should be muuuch higher.